src/components/Charts/allstates.js

```diff
@@ -94,7 +70,7 @@ function AllStatesChart(props) {
     }
 
     datasets.push({
-      borderWidth: 1.5,
+      borderWidth: 3,
```

Comment:
it looks good to me but some felt its will be good to have thin lines. @sudevschiz @shuklaayush @JunaidBabu thoughts?

Reply:
I found the updated colour palette for these lines to not work very well with very thin lines. But if you think the thinner ones looked better, I've updated it to `2` now.